.github/workflows/check-diamond-storage.yml

```diff
@@ -0,0 +1,38 @@
+name: Diamond Storage Check
+
+on:
+  pull_request:
+
+jobs:
+  check-diamond-storage:
+    runs-on: ubuntu-latest
```

Comment:
let's work with the CI, https://github.com/ubiquity/ubiquity-dollar/actions/runs/5476516361/workflow#L10, specific ubuntu version